src/applications/ezr/utils/helpers/form-config.js

```diff
@@ -47,6 +48,41 @@ export function includeTeraInformation(formData) {
   return formData.hasTeraResponse;
 }
 
+export function veteranBornBetween(formData, date1, date2) {
```

Comment:
I'm curious why this method is necessary? The Veteran DOB cannot be before 1900 by default. You could just use an `isBefore` or `isAfter` method in your checks and reduce the complexity of these checks.

Reply:
Okay, I can clean this up!

Reply:
I removed this method and just used the `isBefore` function instead!